_partials/ubuntu_vscode.md

```diff
@@ -0,0 +1,83 @@
+
+## Visual Studio Code - Your text editor
+
+A text editor is one of the most important tools of a developer.
+Follow these instructions in the Terminal:
+
+```bash
+sudo snap install --classic code
+```
+
+:point_up: This command will ask for your password with: `[sudo] password for <username>:`. Don't panick! Calmly type your password key by key. You won't have a visual feedback (like little `*`), that's **perfectly normal**, keep on typing. When you're done, hit `Enter` :muscle:.
+
+In case you don't have `snap` installed - ask a TA for help.
+
+
+Now let's try to launch your VSCode from **the Terminal**:
+
+```bash
+code
+```
+
+If the text editor opens in a new window, you can move forward!
+
+### VS Code Shortcuts
+
+In VS Code:
+
+>\- Click on `File`
+>\- Click on `Preferences`
+>\- Click on `Keymaps`
+>\- Click on `Sublime Text Keymap and Settings Importer`
+>\- Click on `Install`
```

Comment:
This is not rendered as a list: 

![image](https://user-images.githubusercontent.com/37899667/112107903-d68c7900-8baf-11eb-8925-cda830872f72.png)

To render this as a list, you need to add two space at the end of each line. 

BTW, I've seen other `>\-` lists that are not rendered as HTML lists in the setup, even in files not in this PR. Can you make a quick search and replace? :pray:

Reply:
Actually just replacing with simple `-` will do:
![image](https://user-images.githubusercontent.com/39556343/112123069-350e2300-8bc1-11eb-9916-7e90b4e43e66.png)